lib/_http_outgoing.js

```diff
@@ -657,12 +657,11 @@ OutgoingMessage.prototype._flushOutput = function _flushOutput(socket) {
   var output = this.output;
   var outputEncodings = this.outputEncodings;
   var outputCallbacks = this.outputCallbacks;
-  socket.cork();
+
   for (var i = 0; i < outputLength; i++) {
     ret = socket.write(output[i], outputEncodings[i],
                        outputCallbacks[i]);
   }
-  socket.uncork();
```

Comment:
I removed it just to keep the code clean, but do you think it'd be worth leaving in for whoever calls the function without `.write()`? As you said, they don't have an effect.